src/client/Client.js

```diff
@@ -310,6 +312,33 @@ class Client extends BaseClient {
     });
   }
 
+  /**
+   * Obtains a sticker from Discord.
+   * @param {Snowflake} id The sticker's ID
+   * @returns {Promise<Sticker>}
+   * @example
+   * client.fetchSticker('id')
+   *   .then(sticker => console.log(`Obtained sticker with name: ${sticker.name}`))
+   *   .catch(console.error);
+   */
+  async fetchSticker(id) {
+    const data = await this.api.stickers(id).get();
+    return new Sticker(this, data);
+  }
+
+  /**
+   * Obtains the list of sticker packs available to Nitro subscribers from Discord.
+   * @returns {Promise<Collection<Snowflake, StickerPack>>}
+   * @example
+   * client.fetchNitroStickerPacks()
+   *   .then(packs => console.log(`Available sticker packs are: ${packs.map(pack => pack.name).join(', ')}`))
+   *   .catch(console.error);
+   */
+  async fetchNitroStickerPacks() {
+    const data = await this.api('sticker-packs').get();
+    return new Collection(data.sticker_packs.map(p => [p.id, new StickerPack(this, p)]));
```

Comment:
perhaps use reduce? just to be more consistent

Reply:
imo this is cleaner than reduce, and Vlad said that he thinks it's fine

maybe the ones that use reduce to fill a collection can be changed to map instead

cc @iCrawl @kyranet

Reply:
I don't recall if this was one of the patterns we switched from when moving towards reduce, or if it was never written like that.

Either way, I'm fine with this pattern.